include/recognition/planar/HopcroftTarjan.hpp

```diff
@@ -0,0 +1,148 @@
+/*
+ * IPlanarGraphRecognition.hpp
+ *
+ *  Created on: 24.03.2024
+ *      Author: Dzianis Lahunou
+ *      Ported by: Krzysztof Turowski (krzysztof.szymon.turowski@gmail.com)
+ */
+
+#pragma once
+
+#include "recognition/planar/IPlanarGraphRecognition.hpp"
+
+namespace Koala {
+
+const int left = 1;
+const int right = 2;
+/**
+ * @ingroup recognition
+ * The Interface for class for recognition of planar graphs p.
+ *
+ */
+
+class Block {
```

Comment:
Why don't you move this to HopcroftTarjan.cpp? It's a local structure, no need to expose it to anyone using hpp header.